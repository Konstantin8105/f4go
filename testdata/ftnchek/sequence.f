C tests message about statement out of order.
      data x /1.0/
      integer a,b,c
      common a,b,c
      sqr(y)=y*y
      double precision d
      read(*,*) a,b,c,d
      write(*,*) a,b,c,d,sqr(x)
      end
