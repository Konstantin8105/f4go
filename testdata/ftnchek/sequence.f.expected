COMMENT             	C tests message about statement out of order.
NEW_LINE            	

IDENT               	data
IDENT               	x
/                   	/
FLOAT               	1.0
/                   	/
NEW_LINE            	

INTEGER             	integer
IDENT               	a
,                   	,
IDENT               	b
,                   	,
IDENT               	c
NEW_LINE            	

IDENT               	common
IDENT               	a
,                   	,
IDENT               	b
,                   	,
IDENT               	c
NEW_LINE            	

IDENT               	sqr
(                   	(
IDENT               	y
)                   	)
=                   	=
IDENT               	y
*                   	*
IDENT               	y
NEW_LINE            	

IDENT               	double
IDENT               	precision
IDENT               	d
NEW_LINE            	

IDENT               	read
(                   	(
*                   	*
,                   	,
*                   	*
)                   	)
IDENT               	a
,                   	,
IDENT               	b
,                   	,
IDENT               	c
,                   	,
IDENT               	d
NEW_LINE            	

IDENT               	write
(                   	(
*                   	*
,                   	,
*                   	*
)                   	)
IDENT               	a
,                   	,
IDENT               	b
,                   	,
IDENT               	c
,                   	,
IDENT               	d
,                   	,
IDENT               	sqr
(                   	(
IDENT               	x
)                   	)
NEW_LINE            	

END                 	end
NEW_LINE            	

